forge-gui/res/cardsfolder/upcoming/grell_philosopher.txt

```diff
@@ -0,0 +1,12 @@
+Name:Grell Philosopher
+ManaCost:2 U
+Types:Creature Horror Wizard
+PT:1/4
+K:ManaConvert:U->AnyColor:You may spend blue mana as though it were mana of any color to pay the activation costs of CARDNAME's abilities.
```

Comment:
I think this keyword needs to be applied as part of the triggered effect...

@tool4ever @Hanmac what do you think? Obviously it needs to just be a static anyway and not a keyword but that is probably another PR

Reply:
Either way, those would require engine changes that are beyond the scope of my current abilities.

Reply:
Should I just delete the card from the PR and possibly create a new issue? Or maybe edit the current one?

Reply:
> Should I just delete the card from the PR and possibly create a new issue? Or maybe edit the current one?

Let's see what @Hanmac and @tool4ever think

Reply:
The problem here is that the mana conversion restriction is very specific, but the mana conversion as you have it set up will let you spend blue mana to cover costs of just about anything. I think your earlier suggestion of removing this card from this PR for now and/or opening an issue is the right idea.

Reply:
Most of the code for it was already in place, so I made a small fix for it :)

Reply:
Thankee-sai.